cv32e40s/tests/programs/custom/cov_holes_generic/cov_holes_generic.c

```diff
@@ -469,6 +474,28 @@ int cvprintf(verbosity_t verbosity, const char *format, ...) __attribute((__noin
  */
 void vp_assert_irq(uint32_t mask, uint32_t cycle_delay);
 
+/*
+ * test_lfsr_lockup
+ *
+ * Test of LFSR lockup.
+ */
+void test_lfsr_lockup(void);
```

Comment:
The added tests do not use the generic test-template structure, and does not correspond to the comments in the template. This also causes the standard reporting mechanism not to work as intended. Additionally, there is no way to disable the immediate test-termination on error.

Reply:
I realized we already have a test for xsecure csrs (`xsecure_csrs`) so I moved it there.